styles.css

```diff
@@ -0,0 +1,663 @@
+h1 {
+  color: #44BE2F;
+  padding-top: 26%;
+  padding-left: 3rem;
+  font-weight: 500;
+  font-style: italic;
+}
+
+.header-logo {
+  text-align: left;
+  position: absolute;
+  top: 2rem;
+  left: 5rem;
+}
+
+.header-checkout {
+  text-align: right;
+  position: absolute;
+  top: 1em;
+  right: 2%;
+  font-family: monospace;
+  font-weight: 700;
+  font-size: 1em;
+  text-decoration: none;
+  color: #fff;
+}
+.header {
+  height: 20em;
+  background-position: center;
+  background-repeat: no-repeat;
+  background: linear-gradient(#e66465, #9198e5);
+  background-size: cover;
+  position: relative;
+  background-image: url('images/header-image.jpg');
+  overflow: hidden;
+  width: 100%;
+}
+
+
+.header-checkout img {
+  padding-right: 0;
+}
+
+.checkout-img {
+  width: 1.5em;
+}
+
+
+.header-checkout a:link {
+  text-decoration: none;
+  padding: .5em 1em .5em 1em;
+  background-color: #9bc344;
+  border-radius: .5em;
+  border: none;
+  width: 20em;
+  color: #fff;
+}
+
+.header-checkout a:visited {
+  color: #fff;
+  background-color: #9bc344;
+}
+
+.header-checkout a:hover {
+  color: #fff;
+  background-color: #6a872c;
+}
+
+.header-checkout a:active {
+  color: #fff;
+  background-color: #103861;
+}
+
+
+.header-checkout img {
+  padding-right: 0;
+}
+
+
+.dog-pic {
+  width: 20em;
```

Comment:
Does not seem to scale correctly, instead consider using % so it is sized relative to the container. 

`Width: 100%;`

Consider something similar for fixing your button placement.

Reply:
Changed in PC document, will update later.